data/sql/updates/pending_db_world/rev_1601870525065182700.sql

```diff
@@ -0,0 +1,436 @@
+(33701, 6, 0, 'Yogg-Saron is here? It sounds like we really will have our hands full then.', 12, 0, 100, 0, 0, 0, 33713, 0, 'High Explorer Dellorah to Player'),
+(33701, 7, 0, 'What... What did you just do, $N?! Brann! Braaaaannn!', 14, 0, 100, 0, 0, 0, 34446, 0, 'High Explorer Dellorah to Player'),
+(33701, 8, 0, 'Brann! $N just activated the orbital defense system! If we don\'t get out here soon, we\'re going to be toast!', 14, 0, 100, 0, 0, 0, 34447, 0, 'High Explorer Dellorah to Player');
+
+DELETE FROM `gossip_menu_option` WHERE (`MenuID` = 10366);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
+(10366, 0, 0, 'Activate secondary defensive systems.', 34420, 1, 1, 0, 0, 0, 0, NULL, 0, 0);
+
+DELETE FROM `waypoints` WHERE `entry`=33701;
+INSERT INTO `waypoints` (`entry`, `pointid`, `position_x`, `position_y`, `position_z`, `point_comment`) VALUES
+(33701, 1, -767.794, -126.79, 429.83, 'High Explorer Dellorah'),
+(33701, 2, -767.925, -97.062, 429.83, 'High Explorer Dellorah'),
+(33701, 3, -757.399, -75.726, 429.83, 'High Explorer Dellorah'),
+(33701, 4, -742.995, -59.801, 429.83, 'High Explorer Dellorah'),
+(33701, 5, -729.309, -50.896, 429.83, 'High Explorer Dellorah'),
+(33701, 6, -720.001, -53.098, 429.83, 'High Explorer Dellorah'),
+(33701, 7, -723.54, -53.823, 429.83, 'High Explorer Dellorah'),
+(33701, 8, -746.544, -62.526, 429.83, 'High Explorer Dellorah'),
+(33701, 9, -815.776, -128.782, 429.82, 'High Explorer Dellorah'),
+(33701, 10, -856.608, -134.5, 458.89, 'High Explorer Dellorah'),
+(33701, 11, -899.84, -140.921, 458.89, 'High Explorer Dellorah'),
+(33701, 12, -917.893, -143.652, 464.3, 'High Explorer Dellorah');
+
+-- Hired Enginner
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 33626;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = -136348);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-136348, 0, 0, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, -792.043, -44.357, 429.84, 2.41935, 'Hired Engineer - On Data Set 1 1 - Move To Position'),
+(-136348, 0, 1, 0, 60, 0, 100, 1, 0, 0, 0, 0, 0, 62, 603, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, -792.043, -44.357, 429.84, 2.41935, 'Hired Engineer - On Update - Teleport (No Repeat)');
+
+DELETE FROM `conditions` WHERE (`SourceEntry` = -136348);
```

Comment:
missing `SourceTypeOrReferenceId` and `SourceGroup` from `DELETE` clause, this might delete unwanted conditions